dvc/command/data_sync.py

```diff
@@ -116,7 +116,8 @@ def shared_parent_parser():
         "targets",
         nargs="*",
         default=None,
-        help="Limit the scope to these stage files.",
+        help="""Limit the scope to these DVC-files.
+        With -R a directory to search DVC-files in can be specified.""",
```

Comment:
this way it probably means that this line now includes the newline. It's probably not what we want. Have you run the pre-commit hook on the previous patch?

Reply:
@shcheklein, I tried it every possible way. it is the only way to add both these statements in the help or else the build in Travis will fail as adding these two statements will exceed char limit that is 79 and in this 109. Will it be okay if I change the language of this message?

Reply:
have you tried to install and run the pre-commit back hook like described here https://dvc.org/doc/user-guide/contributing#development-environment ? It takes care automatically and split the long string properly for you. I highly recommend you to setup the proper dev environment. Alternatively, take a look a the dvc gc command help - there are some long help messages examples.

Reply:
@shcheklein, Yes I tried them still they were of no use. Check out this screenshot.
![Screenshot (20)](https://user-images.githubusercontent.com/23209383/59459531-9f196600-8e3a-11e9-8cd9-647ac5a1bbe5.png)

Reply:
@sanidhyamangal pre-commit hook won't split string lines for you, you need to do that yourself. As @shcheklein suggested, take a look at gc, where it has long help messages and format this one in a similar way.

Reply:
@efiop  I have updated the requested changes please review them.